devito/core/gpu_openmp.py

```diff
@@ -263,6 +341,7 @@ def _normalize_kwargs(cls, **kwargs):
         o['par-chunk-nonaffine'] = oo.pop('par-chunk-nonaffine', cls.PAR_CHUNK_NONAFFINE)
         o['par-dynamic-work'] = np.inf  # Always use static scheduling
         o['par-nested'] = np.inf  # Never use nested parallelism
+        o['gpu-direct'] = oo.pop('gpu-direct')
```

Comment:
```
oo.pop('gpu-direct', False)
```

otherwise you get an error if the user doesn't provide the option explicitly...

... because (EDIT) you will have to remove the default from operator.py . See comment below